lib/assessment.js

```diff
@@ -8,38 +8,57 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
-
+var min = function min(previousValue, currentValue, index, array) {
+  return currentValue > previousValue ? currentValue : previousValue;
```

Comment:
This is actually `max`, not `min`. It selects for the greater element; used with `Array.reduce`, it will get you the greatest element in the list. Half credit.